src/Restaurant.js

```diff
@@ -1,14 +1,25 @@
-import React from "react";
+import React, { useState } from "react";
+import Order from "./Order";
 
 const Restaurant = () => {
-  const pizzas = 0;
+  const [orders, setOrders] = useState(0);
+  function orderOne() {
+    return setOrders(orders + 1);
+  }
```

Comment:
remember to remove unnecessary state & functions when they're no longer being used, same for commented out code